.github/workflows/ci-maven.yaml

```diff
@@ -156,20 +156,24 @@ jobs:
       matrix:
         java: [ 11, 17 ]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: actions/setup-java@v3
         with:
           java-version: ${{ matrix.java }}
           distribution: temurin
-      - run: java -version
+      - run: mvn -version
       - name: Install maven modules
         run: |
           mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip
 
-      - name: Install system dependencies
+      - name: Showcase golden tests
+        working-directory: showcase
         run: |
-          sudo apt-get update
-          sudo apt-get install -y curl unzip
+          mvn test \
+            -P enable-golden-tests \
+            --batch-mode \
+            --no-transfer-progress
```

Comment:
Do you really need maven that runs bazel that runs a shell script?

Reply:
Showcase has been set up to be invoked with all `mvn` commands to avoid needing to change developer workflows once @blakeli0 has completed https://github.com/googleapis/gapic-generator-java/issues/1069

If this is something you'd like to revisit, I would request that we open a ticket and handle it separately.

Reply:
Fair enough.

Reply:
FYI, I just closed https://github.com/googleapis/gapic-generator-java/issues/1069 as the generator process is already using the gapic-generator-java jar. 
I agree with @suztomo in long term, but since we are still in the middle of mono repo migration cleanup/enhancement, I think this is good for now and we can revisit it once everything is stable.

Reply:
Tracking with new issue: https://github.com/googleapis/gapic-generator-java/issues/1282